app/controllers/sessions_controller.rb

```diff
@@ -6,9 +6,9 @@ def create
 
     if user&.authenticate(params[:session][:password])
       log_in user
-      flash.now[:success] = t ".success"
+      flash[:success] = t ".success"
```

Comment:
login in thành công rồi để người dùng đơ ở đó hử em

Reply:
Dạ cái này Thảo làm giao diện admin nhưng mà pull đó chưa đưuọc merged nên là em đợi merged, rebase rồi redirect đến đó sau ạ